### PR TITLE
Update mvdtool versions

### PR DESCRIPTION
**Pull request by st4rl3ss**
Simple update of MVDtool versions.

### CODE REVIEW

**Comment by st4rl3ss** [comment]
please retest

**Comment by st4rl3ss** [comment]
retest this please

**Comment by pramodk** [comment]
> retest this please

@st4rl3ss : look at the failure logs in the[ tests tab](https://bbpcode.epfl.ch/ci/blue/organizations/jenkins/hpc.spack-deployment.pull-requests/detail/hpc.spack-deployment.pull-requests/2421/tests) where we have:

```
-- Configuring done
CMake Error: install(EXPORT "MVDToolTargets" ...) includes target "MVDTool" which requires target "Sonata" that is not in the export set.
-- Generating done
CMake Generate step failed. Build files cannot be regenerated correctly.
```

May be this is something relevant error?

**Comment by st4rl3ss** [comment]
I must have looked wrong since before I saw a more generic error.
If this is the error that appears then it should be solved when my PR on MVDTool passes. I will retry once that is merged.
Thanks for pointing this out.

File: var/spack/repos/builtin/packages/py-mvdtool/package.py [hunk 1]
```diff
@@ -15,6 +15,9 @@ class PyMvdtool(PythonPackage):
     git      = "https://github.com/BlueBrain/MVDTool.git"
 
     version('develop', branch='master', submodules=True, get_full_repo=False)
+    version('2.4.2', tag='v2.4.2', submodules=True, get_full_repo=False)
+    # Version 2.4.1 might have some issue in installing via Spack
+    # Please use a more recent version.
```
Comment (matz-e) [inline var/spack/repos/builtin/packages/py-mvdtool/package.py]: Then just drop it :D

Comment (st4rl3ss) [reply var/spack/repos/builtin/packages/py-mvdtool/package.py]: Makes sense!

**Review by sergiorg-hpc — APPROVED** [review]

**Pull request merged**